src/components/ProfilePage.jsx

```diff
@@ -44,6 +44,7 @@ export class ProfilePage extends React.Component {
     this.props.fetchProfile(this.props.match.params.username);
     logEvent('edx.profile.viewed', {
```

Comment:
Reviewers note: I could also temporarily change to `edx.bi.profile.viewed` in addition to or in place of the `version`, to better match our legacy practices for now.

Reply:
I sympathize with the desire to get a test event out there.  I think we should discuss versioning more before rolling it out, because I would prefer it to be more systematic than ad hoc.  

I prefer the name-change suggestion if this is a place-holder event, and the edx.bi.profile.viewed suggestion as a legacy practice seems good to me.   I would also be in favor of something like "edx.dev.profile.viewed", where we can filter those events from export to researchers based on the prefix "edx.dev" (or a similar convention), but we don't have that in place now and want that to be part of the versioning conversation we have.  (It's not in place for the "edx.bi" events either, but that's more entrenched as a legacy practice. :)

Also, the spec lists the property as "profile_username", but the property below is "profileUsername".  I think we should use the underscore format for greater consistency with the properties of existing events, all else being equal.  That said, I think that just "username" would also be adequate.

Reply:
FYI: The JavaScript camelcase is automatically converted to snakecase before the event gets sent.